### PR TITLE
Adjust voice

### PR DESCRIPTION
**Pull request by Pryadka**
This change aligns voices in seconds as it requires here https://github.com/eNote-GmbH/notengrafik/blob/master/testset/voices-in-seconds_expected.svg . Probably 0-margin is not perfect and it needs to be adjusted.

### CODE REVIEW

**Comment by craigsapp** [comment]
https://github.com/eNote-GmbH/notengrafik/blob/master/testset/voices-in-seconds_expected.svg

Perhaps that is in a private repository?, as I cannot see it.

**Comment by rettinghaus** [comment]
It is indeed. 
@Pryadka you should add a MEI test file and the Verovio output.

File: include/vrv/object.h [hunk 1]
```diff
@@ -1132,7 +1132,7 @@ class Object : public BoundingBox {
     /**
      * Initialisation method taking a uuid prefix argument.
      */
-    void Init(std::string);
+    void Init(const std::string&);
```
Comment (lpugin) [inline include/vrv/object.h]: Add the missing `classId` name

File: src/bboxdevicecontext.cpp [hunk 2]
```diff
@@ -300,8 +300,9 @@ void BBoxDeviceContext::DrawText(const std::string &text, const std::wstring wte
         m_textX -= extend.m_width;
     }
     else if (m_textAlignment == HORIZONTALALIGNMENT_center) {
-        m_textX -= (extend.m_width / 2);
+        m_textX -= extend.m_width / 2;
```
Comment (lpugin) [inline src/bboxdevicecontext.cpp]: Leave the ()

**Comment by Pryadka** [comment]
Input data file :
[voices-in-seconds.txt](https://github.com/rism-ch/verovio/files/4318384/voices-in-seconds.txt)
Current implementation:
current
Expected:
expected

File: include/vrv/object.h (hunk 1 above)
Comment (Pryadka) [reply include/vrv/object.h]: Done

File: src/bboxdevicecontext.cpp (hunk 2 above)
Comment (Pryadka) [reply src/bboxdevicecontext.cpp]: Done

File: src/layerelement.cpp [hunk 3]
```diff
@@ -1153,6 +1153,11 @@ int LayerElement::AdjustLayers(FunctorParams *functorParams)
                 }
                 else if (abs(previousNote->GetDrawingLoc() - params->m_currentNote->GetDrawingLoc()) > 1)
                     continue;
+                // check if the next note is simultaneous and put in with adjustable space
```
Comment (lpugin) [inline src/layerelement.cpp]: I am not sure how this works. A note will always have a staff ancestor. When do you expect this not to be true? What is the logic of the change?
Also, you do not need to do a dynamic_cast to compare with a NULL.  (Please change `nullptr` to `NULL` for consistency)

Comment (Pryadka) [reply src/layerelement.cpp]: Sorry, I didn't know about staff ancestors. I removed this check.

**Pull request closed without merging**